Automattic/wp-calypso
# Signup: Remove domain pre-filling after negative A/B test

Manual revert of #22376

As explained in recent comments to p7jreA-1vS-p2, the A/B test regarding domain pre-filling had negative results, favoring the established behaviour.

This PR removes everything that was introduced in #22376, as it was not possible to revert it automatically anymore.

## Testing instructions

Start the signup flow from scratch (typically on `/start/about`).
Fill in the site title and proceed to next step.
The domain field should **not** be populated with the site title, and there should **not** be any search results displayed.
